### DISCUSSION TITLE
Alarm trigger PowerShell script?

### DISCUSSION THREAD

**Discussion by LetThatBotIn** [homebridge-simplisafe3/homebridge-simplisafe3#233] [Q&A]
Hello,

I’ve got the SS plug in installed on my Win10 machine (and although it doesn’t really help for what I’m trying to do, it is linked to HomeKit). I’m trying to figure out if it’s possible to have a PowerShell script (or VBS if I must) run when an alarm is triggered? 

Thanks,

**Comment by shamoon** [comment]
I have no idea but I think your question is a bit more general than this plugin, its to do with getting Powershell integrated with homebridge, so maybe https://www.google.com/search?q=homebridge+powershell ?

Also, I wonder about something like node-red or even HomeAssistant, depending on your needs.

Good luck
